timeless/reservations/views.py

```diff
@@ -94,19 +94,22 @@ def get(self, company_id):
 
 @bp.route("/")
 def list_reservations(reservations):
-    """
-        @todo #172:30min Refactor this after the implementation of GenericViews.
-         Take a look at puzzles #134 and #173 where the requirements of generic
-         views are described. Don't forget to cover the generated code with
-         tests
-
-    :param reservations:
-    :return:
+    reservations_result = []
```

Comment:
@ebercamargo We have a `ListView` generic class, pls use it here. Look at code base there are a lot of examples.